cmd/monitor/monitor_describe.go

```diff
@@ -0,0 +1,67 @@
+package monitor
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"gitlab.com/stackvista/stackstate-cli2/cmd/settings"
+	sts "gitlab.com/stackvista/stackstate-cli2/generated/stackstate_api"
+	"gitlab.com/stackvista/stackstate-cli2/internal/common"
+	"gitlab.com/stackvista/stackstate-cli2/internal/di"
+	"gitlab.com/stackvista/stackstate-cli2/internal/util"
+)
+
+type DescribeArgs struct {
+	ID       int64
+	FilePath string
```

Comment:
I wouldn't introduce this option. Printing to stdout should be enough. Rest can be done by unix piping.

Reply:
It's consistent with `settings describe` though.